Python/Tim sort.py

```diff
@@ -0,0 +1,41 @@
+def timsort(array):

```

Comment:
Mention Time Complexity

Reply:
It has O(n*logn) time complexity for worst case and O(n) for best case scenarios.

Reply:
Add that in the program and commit